### ISSUE TITLE
[image caption] saved models have higher loss 

### ISSUE THREAD

**Issue by laptopmutia** [yunjey/pytorch-tutorial#130]
when I load the saved model and resume the training
the loss is always higher than it supposed to be

I also have saved the optimizer but the result is same 
the resumed training always have higher loss than it supposed to be

anyone know what make this happen? and how to fix it?

**Comment by yunjey** [comment]
The provided checkpoints were saved at intermediate step, not final step. Therefore, it can have higher loss than the final checpoint.
